libs/service-portal/occupational-licenses/src/screens/HealthDirectorateDetail/HealthDirectorateDetail.tsx

```diff
@@ -0,0 +1,90 @@
+import { useParams } from 'react-router-dom'
+import { useGetHealthDirectorateLicenseByIdQuery } from './HealthDirectorateDetail.generated'
+import { Box } from '@island.is/island-ui/core'
+import { CardLoader, ErrorScreen } from '@island.is/service-portal/core'
+import { useLocale } from '@island.is/localization'
+import { useUserInfo } from '@island.is/auth/react'
+import { getOrganizationLogoUrl } from '@island.is/shared/utils'
+import { Organization } from '@island.is/shared/types'
+import { LicenseDetail } from '../../components/LicenseDetail'
+import { olMessage as om } from '../../lib/messages'
+import { m } from '@island.is/service-portal/core'
+
+type UseParams = {
+  id: string
+}
+
+export const EducationDetail = () => {
+  const { id } = useParams() as UseParams
+
+  const user = useUserInfo()
+  const birthday = user.profile.dateOfBirth ?? null
+
+  const { formatDateFns, formatMessage } = useLocale()
+
+  const { data, loading, error } = useGetHealthDirectorateLicenseByIdQuery({
+    variables: {
+      id: id,
+    },
+  })
+
+  const license = { ...data?.occupationalLicenseHealthDirectorateLicense }
+
+  console.log(license)
+
+  if (loading)
+    return (
+      <Box paddingTop="p1">
+        <CardLoader />
+      </Box>
+    )
+
+  if (
+    error ||
+    !license.name ||
+    !license.profession ||
+    !license.license ||
+    !license.licenseNumber ||
+    !license.validFrom ||
+    !birthday
+  )
+    return (
+      <ErrorScreen
+        title={formatMessage(om.errorFetchLicense)}
+        tag={formatMessage(m.errorFetch)}
+      />
+    )
```

Comment:
Seems pretty hardcore to return an error if your health directorate license is only missing a birthday. 
This check is a bit draconian